src/mem.rs

```diff
@@ -0,0 +1,60 @@
+// NOTE Copied verbatim from the rlibc crate
```

Comment:
The focus of this crate is to provide all of the runtime functions that are required to run Rust with libcore. This includes both compiler-rt builtins and a few libc functions (memset, memmove, memcpy, memcmp, fmod, fmodf).